lms/templates/instructor/instructor_dashboard_2/discussions.underscore

```diff
@@ -1,5 +1,26 @@
 <!-- Discussion Topics. -->
 <div class="discussions-nav" id="discussions-management" tabindex="-1">
-    <div class="course-wide-discussions-nav"></div>
-    <div class="inline-discussions-nav"></div>
+    <div class="hd hd-3 subsection-title"><%- gettext('Specify whether discussion topics are divided') %></div>
+    <div class="division-scheme-container">
+        <div class="division-scheme-items">
+            <% for (var i = 0; i < availableSchemes.length; i++) { %>
```

Comment:
I believe this error is related:

![image](https://cloud.githubusercontent.com/assets/484484/26697947/e7c20198-46e1-11e7-8fda-78794389ddd0.png)